runtime/pom.xml

```diff
@@ -54,15 +54,15 @@
             f64_cmp.wast,
             float_misc.wast,
             fac.wast,
-            data.wast,
```

Comment:
this test was erroneously passing only because it was never instantiated.
Now the logic is fixed in `test-gen`.